docs2017/src/api/cloudant-geo.md

```diff
@@ -271,7 +271,7 @@ _	Result set.
 _Example format for a Cloudant Geo API call:_
 
 ```http
-/<database>/_design/<name>/_geo/<geoindexname>?<query-parameters>
+/$DATABASE/_design/$ACCOUNT/_geo/<geoindexname>?<query-parameters
```

Comment:
`<name>` here should not be replaced with `$ACCOUNT`. Suggest using `$DDOC` as the standard template for design document name. Also replace `<geoindexname>` with a similar template, making the whole thing: `/$DATABASE/_design/$DDOC/_geo/$INDEX_NAME?$QUERY_PARAMS`